agent/util-scripts/tool-meister/pbench-sysinfo-dump

```diff
@@ -1,6 +1,24 @@
 #!/bin/bash
 # -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; tab-width: 8 -*-
 
+# Installation directory.
+install_dir="$(realpath -e ${sysinfo_install_dir} 2> /dev/null)"
```

Comment:
Is there a specific reason to use `install_dir` instead of `pbench_install_dir`? I thought since this is pbench specific install directory, we would have pbench in its name

Reply:
It doesn't matter here since it's local to the bash script; it's not `export`ed anywhere. Our code would be hard to read if we used `_pbench_` prefixes for *every* local variable. (We actually did that in threads, which was a nightmare, and I really enjoyed getting everyone to tear that out when we went "pthreads native".)

Reply:
> It doesn't matter here since it's local to the bash script; it's not `export`ed anywhere

That make sense